src/systemctl/systemctl-logind.c

```diff
@@ -152,6 +152,8 @@ int logind_check_inhibitors(enum action a) {
                 return 0;
 
         r = acquire_bus(BUS_FULL, &bus);
+        if (r == -ECONNREFUSED && geteuid() == 0)
+                return 0; /* dbus.service seems not running. */
```

Comment:
Maybe a more explicit comment such as: `When D-Bus is not running, allow root to force a shutdown. E.g.: when running at the emergency console."